extension/chrome/elements/shared/key_errors.ts

```diff
@@ -0,0 +1,106 @@
+/* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
+
+'use strict';
+
+import { BrowserMsg } from '../../../js/common/browser/browser-msg.js';
+import { Ui } from '../../../js/common/browser/ui.js';
+import { ClientConfiguration } from '../../../js/common/client-configuration.js';
+import { Key, UnexpectedKeyTypeError } from '../../../js/common/core/crypto/key.js';
+import { saveKeysAndPassPhrase, setPassphraseForPrvs } from '../../../js/common/helpers.js';
+import { Lang } from '../../../js/common/lang.js';
+import { Catch } from '../../../js/common/platform/catch.js';
+import { Settings } from '../../../js/common/settings.js';
+import { UserAlert, KeyCanBeFixed } from '../../../js/common/ui/key-import-ui.js';
+import { SetupOptions, SetupView } from '../../settings/setup.js';
+
+export class KeyErrors {
+  protected fesUrl: string;
+  private readonly acctEmail: string;
+  private readonly parentTabId: string;
+  private readonly clientConfiguration: ClientConfiguration;
+  private readonly setupView?: SetupView;
```

Comment:
It seems these properties are used only in this class, so we don't need `protected` and `readonly`, they all can be just `private`.

Also let's simplify properties and constructor code by using [parameter properties](https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties):
```ts
public constructor(
    private fesUrl: string,
    private acctEmail: string,
    private parentTabId: string,
    private clientConfiguration: ClientConfiguration,
    private setupView?: SetupView
) {}
```

Reply:
Thank you very much. I have learned more here. I have removed the unused properties and converted them into parameter properties.